x-pack/legacy/plugins/apm/public/components/app/ErrorGroupDetails/Distribution/index.tsx

```diff
@@ -79,7 +82,15 @@ export function ErrorDistribution({ distribution, title }: Props) {
       <Histogram
         tooltipHeader={tooltipHeader}
         verticalLineHover={(bucket: FormattedBucket) => bucket.x}
-        xType="time"
+        xType="time-utc"
+        formatX={(value: Date) => {
+          const time = value.getTime();
+          const xMin = d3.min(buckets, d => d.x0);
+          const xMax = d3.max(buckets, d => d.x);
+          return scaleUtc()
+            .domain([xMin, xMax])
+            .tickFormat()(new Date(time - getTimezoneOffsetInMs(time)));
```

Comment:
Up until `tickFormat` can be moved out of this `formatX` function and just created every time `buckets` change. Would be better for performance without too much of an impact on readability I think. Thoughts?